book/src/requirements.md

```diff
@@ -0,0 +1,170 @@
+# Requirements
+
+The design of Pollard aimed to be mostly identical to the Sapling circuit in
+functionality. We used this opportunity to clarify the requirements for Pollard at
+the product and engineering levels.
+
+## Protocol Product Requirements
+
+### ProtoPR1. Pollard Addresses
```

Comment:
Shouldn't the first Protocol Product Requirement be "Eliminate the Need for a Trusted Setup"?

Reply:
Yes, agreed. I believe this is strongly implied by some of the rationales, but I agree it's a foremost requirement.

Reply:
I've added a requirement to spell this out. Please see the `ProtoPR1` in revision 92ed81b.